scripts/push.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "travis@travis-ci.org"
+  git config --global user.name "Travis CI"
+  git remote set-url origin  https://${GITHUB_TOKEN}@github.com/rajdas98/community-charts.git > /dev/null 2>&1
+}
+
+commit_website_files() {
```

Comment:
Change the function name to: commit_website_files() -> `commit_updated_changes()`